runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttConditionConfigAdapter.java

```diff
@@ -76,6 +79,15 @@ public JsonObject adaptToJson(
             {
                 JsonObjectBuilder subscribeJson = Json.createObjectBuilder();
                 subscribeJson.add(TOPIC_NAME, s.topic);
+                if (!s.params.isEmpty())
+                {
+                    JsonObjectBuilder params = Json.createObjectBuilder();
+                    s.params.forEach(par ->
+                    {
+                        params.add(par.name, par.value);
+                    });
```

Comment:
```suggestion
                    sub.params.forEach(p-> params.add(p.name, p.value));
```
Use `sub` instead of `s` in `mqttCondition.subscribes.forEach`.